File: Unidad3/Programa5U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa5u3;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class Programa5U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
       double p1,p2,p3,total;
        
        System.out.println("Ingresa el valor 1:");
        p1 = sc.nextDouble();
        System.out.println("Ingresa el valor 2:");
        p2 = sc.nextDouble();
        System.out.println("Ingresa el valor 3:");
        p3 = sc.nextDouble();
        
        total= p1+p2+p3;
        
        if(total >= 1500)
        {
            total = total - (total*0.30); 
            System.out.println("Tiene el 30% de descuento: " + total);
        }else if(total<1500 && total >=1000)
        {
            total = total - (total*0.20); 
            System.out.println("Tiene el 20% de descuento: " + total);
        }else if(total<=1000 && total >=700)
        {
            total = total - (total*0.30); 
            System.out.println("Tiene el 10% de descuento: " + total);
        }else
        {
            System.out.println(total);
        }
    }
}

File: Unidad3/u3.cs/Programa5U3.cs
using System;

namespace Programa5U3;

class descuento
{
    static void Main(string[] args)
    {
        double p1, p2, p3, total;

        Console.WriteLine("Ingresa el valor 1:");
        p1=double.Parse(Console.ReadLine());
        Console.WriteLine("Ingresa el valor 1:");
        p2 = double.Parse(Console.ReadLine());
        Console.WriteLine("Ingresa el valor 1:");
        p3 = double.Parse(Console.ReadLine());

        total = p1+ p2 + p3;

        if (total >= 1500)
        {
            total = total - (total * 0.30);
            Console.WriteLine("Tiene el 30% de descuento: " + total);
        }
        else if (total < 1500 && total >= 1000)
        {
            total = total - (total * 0.20);
            Console.WriteLine("Tiene el 20% de descuento: " + total);
        }
        else if (total <= 1000 && total >= 700)
        {
            total = total - (total * 0.30);
            Console.WriteLine("Tiene el 10% de descuento: " + total);
        }
        else
        {
            Console.WriteLine("Sin descuento"+ total);
        }
    }
}
